OnLand/ControlLaptop/angular-UI/src/app/components/navigation-info/navigation-info.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="navigation-info">
+    <H2>Navigation</H2>
+    <p>Longitude: 210</p>
+    <p>Latitude: 40</p>
+    <p>Heading: 10</p>
+    <p>Depth: -1.3</p>
+    <p>Tunnel Length: 20</p>
```

Comment:
In this case, I think we could try using table tag instead. https://www.w3schools.com/html/html_tables.asp